sdk/keyvault/keyvault-certificates/README.md

```diff
@@ -176,19 +176,17 @@ const client = new CertificateClient(url, credential);
 const certificateName = "MyCertificateName";
 
 async function main() {
-  const keyVaultCertificate = await client.createCertificate(certificateName, {
+  await client.beginCreateCertificate(certificateName, {
     issuerName: "Self",
     subject: "cn=MyCert"
   });
-  console.log("Certificate: ", keyVaultCertificate);
 }
 
 main();
 ```
 
-Besides the name of the certificate, you can also pass the following attributes:
+Besides the name of the certificate and the policy, you can also pass the following properties in a thid argument with unrequired optional values:
```

Comment:
unrequired would mean the same thing as optional values. You could just say "with optional values"